pkg/doc/verifiable/support_test.go

```diff
@@ -206,3 +210,30 @@ func (s rs256TestSigner) Sign(data []byte) ([]byte, error) {
 func publicKeyPemToBytes(publicKey *rsa.PublicKey) []byte {
 	return x509.MarshalPKCS1PublicKey(publicKey)
 }
+
+func createVCWithLinkedDataProof() (*Credential, PublicKeyFetcher) {
+	vc, err := NewUnverifiedCredential([]byte(validCredential))
+	if err != nil {
+		panic(err)
+	}
+
+	created := time.Now()
+
+	pubKey, privKey, err := ed25519.GenerateKey(rand.Reader)
+	if err != nil {
+		panic(err)
```

Comment:
why panic in test?

Reply:
I saw this approach somewhere in aries. It's used when e.g. there is a test utility function which is used in many tests. There are a few options to handle error cases in this utility function:
- write it as production code function - add `error` to the return arguments etc.;
- pass `t *testing.T` and assert using it;
- use panic().

@troyronda what is the best approach from your point of view?